### ISSUE TITLE
Функция доставки уведомлений

### ISSUE THREAD

**Issue by EeOneDown** [hiking-heroes/Backlog#20]
- [x] Server
- [ ] Mobile
- [ ] Integration

Необходимы уведомления:
- [x] Вход в приложение *
- [x] Создание ивента *
- [x] Регистрация на ивент *
- [x] Запрос на участие (при регистрации пользователя) (для владельца) 
- [ ] Инвайт (для участника)
- [ ] Подтверждение участия (для участника)
- [ ] Предупреждение за день до начала

*А надо ли? (@AlexKornShell)

**Comment by EeOneDown** [comment]
Я, кстати, могу организовать рассылку о завтрашнем событии. Но она будет производиться для всех пользователей в одно время один раз в сутки. 
На PA будет запускаться скрипт каждый день в 8 вечера (условно), который будет проверять события у пользователей, который подписаны на рассылку, и если завтра есть событие, то кидать уведомление.
Проблема здесь в том, что событий может быть несколько. И либо придет куча уведомлений, либо я не знаю что делать)

**Comment by AlexKornShell** [comment]
Так. Ну для одного события - это нормально. Чтобы показать работоспособность.

**Comment by AlexKornShell** [comment]
Первые 3 - реализовать проще простого. Их можно показать. 
Последнее - то, что реально от всего этого требуется :)
Но оно и сложно.

**Comment by EeOneDown** [comment]
Не думаю, что очень сложно.
Но у нас нет реализации инвайтов/подтверждений

**Comment by EeOneDown** [comment]
@AlexKornShell 
Вообще, на стороне сервера я могу это сделать к завтра...
